src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -374,47 +376,21 @@ public Boolean visitDeclared(AnnotatedDeclaredType type, Void p) {
         }
     };
 
-    /**
-     * TODO: Similar but not the same as AnnotatedTypeFactory.constructorFromUse with space set aside from
-     * TODO: comb constraints, track down the differences with constructorFromUse
-     * Note: super() and this() calls
-     * @see org.checkerframework.checker.type.AnnotatedTypeFactory#constructorFromUse(com.sun.source.tree.NewClassTree)
-     *
-     * @param newClassTree
-     * @return
-     */
     @Override
     public ParameterizedExecutableType constructorFromUse(final NewClassTree newClassTree) {
         assert newClassTree != null : "NewClassTree was null when attempting to get constructorFromUse. " +
                                       "Current path:\n" + getVisitorTreePath();
 
         final ExecutableElement constructorElem = TreeUtils.elementFromUse(newClassTree);
-        @SuppressWarnings("deprecation") // TODO
-        final AnnotatedTypeMirror constructorReturnType = fromNewClass(newClassTree);
+        // Get constructorReturnType for add equality constraint by calling addComputedTypeAnnotations
```

Comment:
```suggestion
        // Add equality constraints to return type by calling addComputedTypeAnnotations.
```

Why is this only necessary for the return type, not the parameter types? The old code also did something with the parameters.